src/sentry/api/event_search.py

```diff
@@ -1022,6 +1027,19 @@ def has_default(self, params):
         "args": [],
         "transform": "divide(countIf(notEquals(transaction_status, 0)), count())",
     },
+    "histogram": {
```

Comment:
We don't have plans to expose this in the column builder do we?

Reply:
No I would very much like to avoid that.